src/js/jsx/sections/transform/Size.jsx

```diff
@@ -78,6 +139,64 @@ define(function (require, exports, module) {
         },
 
         /**
+         * Saves the current width of selected layers on scrub start
+         *
+         * @private
+         */
+        _handleWScrubBegin: function () {
+            this._initialScrubW = collection.uniformValue(this.state.widths);
+        },
+
+        /**
+         * Update the current width of the selected layers based on change from initial scrub position
+         *
+         * @private
+         * @param {number} deltaX
+         */
+        _handleWScrub: function (deltaX) {
+            if (_.isNumber(this._initialScrubW)) {
+                var document = this.props.document,
+                    uiStore = this.getFlux().store("ui"),
+                    resolvedDelta = uiStore.zoomCanvasToWindow(deltaX),
```

Comment:
I think it's contextual more than general. I wouldn't want the deltaX to be multiplied by any number on rotation scrub or radius scrub for example.